kubenow/gluster-s3/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: "S3 object storage interface for GlusterFS"
+name:  gluster-s3
+version: 1.6.0
```

Comment:
I believe this is the version of the Chart, not the version of the software. Please use 0.1.0 if I am right.